app/components/SideBar/AccountsList/AccountsList.module.css

```diff
@@ -0,0 +1,49 @@
+.sidebarMenuTotalBalanceExtended {
```

Comment:
I would suggest deleting `sidebarMenu` prefix from all classes, I guess they were needed to avoid classe clashes, we don't have such thing with css module :)

Reply:
Of course, I've just renamed them in the last commit.